packages/paste-design-tokens/tokens/global/background-color.yml

```diff
@@ -29,6 +29,12 @@ props:
   color-background-brand:
     value: "{!palette-blue-80}"
     comment: Primary brand background, accessible with inverse text
+  color-background-brand-strong:
+    value: "{!palette-blue-90}"
+    comment: Primary brand background, accessible with inverse text
+  color-background-brand-stronger:
+    value: "{!palette-blue-100}"
+    comment: Primary brand background, accessible with inverse text
```

Comment:
Should we update these comments to be "Strong primary..." and  "Stronger primary..." or "dark" and "darker" like the other token comments?

Reply:
Great shout